pep/app/authenticators/credentials.ts

```diff
@@ -103,6 +105,7 @@ export default class CredentialsAuthenticator extends BaseAuthenticator {
     async invalidate(data: PepSecureAuthenticatedData) {
         try {
             const params = serializeQueryParams({ SessionId: data.SessionId });
+            this.currentUser.clearPreferences();
```

Comment:
why are we clearing preferences on logout, was this in the spec?

based on my past discussions w/ @adistasio, i was under the impression that local preferences (cookies, local storage) should always be preserved, regardless of if the user is logged in or out.

Reply:
Not sure, but I think it makes the most sense. If you log out, we shouldn't be showing your theme choice and other choices you made - we should be showing the default one.

Reply:
yeah so we discussed that a bit, but we came to the conclusion that prefs (local-stored ones at least) should just always stick around. One reason was that you can set preferences like theme and search fields while logged out, and they will be preserved. So would seem odd that if you then happened to log in at some point later, and then logged out, that those prefs would be reset, since they never got reset before as long as you stayed logged out.

Reply:
I still think it might be better to give the users a clean slate. If you share a computer with a colleague and he logs out, you don't want to have to go and change all his settings. That way you have a consistent reproducible logged out state each time @adistasio

Reply:
Let's leave it as is, don't clear preferences on logout.